README.md

```
@@ -1,28 +1,98 @@
 
-## Mentor
-- Tasin-BS
+**Your friendly, all-in-one file management solution.** 
+<br />
+![Banner](public/images/banner.svg)
+</div>
+
+## About Oakcan
+Oakcan is a file management system that allows users to upload, download, and manage files with ease. It provides a user-friendly web interface and robust features to ensure a seamless experience.
+
+<div align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)